apps/src/applab/applab.js

```diff
@@ -350,7 +350,8 @@ function renderFooterInSharedGame() {
     },
     {
       text: applabMsg.privacyPolicy(),
-      link: 'https://code.org/privacy'
+      link: 'https://code.org/privacy',
```

Comment:
It looks like some places I did https and some I did http. Should I just be doing https everywhere at this point?